src/rez/pip.py

```diff
@@ -245,9 +242,13 @@ def pip_install_package(source_name, pip_version=None, python_version=None,
     _system = System()
 
     # Collect resulting python packages using distlib
-    distribution_path = DistributionPath([destpath], include_egg=True)
+    distribution_path = DistributionPath([destpath])
     distributions = [d for d in distribution_path.get_distributions()]
 
+    folders = [folder for folder in os.listdir(destpath) if os.path.isdir(os.path.join(destpath, folder))]
```

Comment:
I would add a comment along the lines of, "moving bin folder to expected relative location, rather than under python/" etc. Also, simpler code would suffice:

```
staged_binpath = os.path.join(destpath, "bin")
if os.path.isdir(staged_binpath):
    shutil.move(...)
```

Reply:
Yes indeed, since we only care about that path no reason to loop. It always seems to be under the same location so no need to recurse the directory for other bin folders. That is what I had in mind initially.